test/functional/test_runner.py

```diff
@@ -258,6 +258,7 @@
     'wallet_implicitsegwit.py --legacy-wallet',
     'rpc_named_arguments.py',
     'feature_startupnotify.py',
+    'wallet_simulaterawtx.py',
```

Comment:
In 84a745eed211b597a89f3bbc3da73b0ac2cb457d "rpc/wallet: add simulaterawtransaction RPC"

Wallet functional tests should appear in test_runner.py twice, once with `--legacy-wallet` and once with `--descriptors`.

Reply:
Got it. Done.